charts/ingress-nginx/templates/default-backend-role.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.rbac.create .Values.podSecurityPolicy.enabled .Values.defaultBackend.enabled -}}
+{{- if and .Values.rbac.create .Values.podSecurityPolicy.enabled (not .Values.global.podSecurityStandards.enforced) .Values.defaultBackend.enabled -}}
```

Comment:
As Giant Swarm is dropping PSP in recent releases and upstream is planning on doing so, that diff will probably disappear in the near future.